mkosi/resources/mkosi-initrd/mkosi.conf.d/20-debian-ubuntu-libtss.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=|debian
+Release=|bullseye
+Distribution=|ubuntu
+Release=|focal
```

Comment:
Ditto

Reply:
Why focal here btw? I thought these libraries didn't exist on focal?

Reply:
Right... Confused that while jotting it down.